adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -495,34 +540,49 @@ public AuthenticationResult acquireTokenSilentSync(String resource, String clien
             throw new IllegalArgumentException("The required clientId is null or blank.");
         }
 
+        final String requestId = Telemetry.registerNewRequest();
+        final APIEvent apiEvent = createApiEvent(mContext, clientId, requestId, EventStrings.ACQUIRE_TOKEN_SILENT_SYNC);
+        apiEvent.setSilentRequestPromptBehavior(PromptBehavior.Auto.toString());
```

Comment:
my bad.

---
In reply to: [90355053](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/736#discussion_r90355053) [](ancestors = 90355053)